javasample/src/main/java/com/oursky/authgeartest/MainActivity.java

```diff
@@ -112,13 +123,19 @@ private void updateButtonDisabledState(MainViewModel viewModel) {
         UserInfo userInfo = viewModel.userInfo().getValue();
         boolean isLoading = viewModel.isLoading().getValue();
         boolean isConfigured = viewModel.isConfigured().getValue();
-        boolean isLoggedIn = viewModel.isLoggedIn().getValue();
         boolean isAnonymous = userInfo != null && userInfo.isAnonymous();
+        boolean isBiometricSupported = viewModel.isBiometricSupported().getValue();
+        boolean isBiometricEnabled = viewModel.isBiometricEnabled().getValue();
+        boolean isLoggedIn = userInfo != null;
```

Comment:
The buttons isLoggedIn state is incorrect. When I 

1. call authorize login success
2. kill the app
3. reopen the app
4. call configure

I should logged in, but the UI show me I am not.

Reply:
The iOS example app has the same behavior. Do you want me to correct all SDK examples?